examples/rtctest/Makefile

```diff
@@ -0,0 +1,62 @@
+PROG_NAME = rtctest
+PROG_TITLE = "RTC Test"
+
+# Paths
+SDK_DIR = $(N64_INST)
+SDK_LIB_DIR = $(SDK_DIR)/mips64-elf/lib
+N64_GCC_PREFIX = $(SDK_DIR)/bin/mips64-elf-
+
+# GCC binaries
+CC = $(N64_GCC_PREFIX)gcc
+CXX = $(N64_GCC_PREFIX)g++
+AS = $(N64_GCC_PREFIX)as
+LD = $(N64_GCC_PREFIX)ld
+OBJCOPY = $(N64_GCC_PREFIX)objcopy
+
+# GCC Flags
+CFLAGS = -std=gnu99 -march=vr4300 -mtune=vr4300 -g -Og -Wall -Werror
+CFLAGS += -I$(SDK_DIR)/mips64-elf/include
+LDFLAGS = --library=dragon --library=c --library=m --library=dragonsys
+LDFLAGS += --library-path=$(SDK_LIB_DIR) --script=n64.ld --gc-sections
+
+# LibDragon binaries
+ROM_HEADER = $(SDK_LIB_DIR)/header
+CHKSUM64 = $(SDK_DIR)/bin/chksum64
+N64TOOL = $(SDK_DIR)/bin/n64tool
+
+# LibDragon Flags
+OUT_SIZE = 1052672B # 52672B HEADER + 1M (minimum) ROM
```

Comment:
This explanation is incorrect. Rather the minimum size is 1 MiB = 1048576 bytes (not decimal megabytes) and the header size is 4096 bytes.

Reply:
You’re right, that was a gnarly workaround for a bug in n64tool that has now been fixed. Good catch.